jupyterhub/postBuild

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```

Comment:
the fact that it is only a `set....` command makes me wonder if this is necessary